docs/using.md

```diff
@@ -5,86 +5,56 @@ overview of different components and services.
+- [Managing Minishift](#managing-minishift)
+  - [Minishift start](#minishift start)
+  - [Minishift stop](#minishift-stop)
+  - [Minishift delete](#minishift-delete)
 - [Minishift runtime options](#minishift-runtime-options)
   - [Flags](#flags)
   - [Environment variables](#environment-variables)
   - [Persistent configuration](#persistent-configuration)
     - [Setting persistent configuration values](#setting-persistent-configuration-values)
     - [Unsetting persistent configuration values](#unsetting-persistent-configuration-values)
+- [HTTP/HTTPS Proxies](#httphttps-proxies)
+- [Mounted host folders](#mounted-host-folders)
+  - [Mounting custom shared folders](#mounting-custom-shared-folders)
+- [Networking](#networking)
 - [Interacting with OpenShift](#interacting-with-openshift)
   - [OpenShift client binary \(oc\)](#openshift-client-binary-oc)
   - [Login](#login)
   - [Console](#console)
   - [Services](#services)
-- [HTTP/HTTPS Proxies](#httphttps-proxies)
-- [Mounted host folders](#mounted-host-folders)
-  - [Mounting custom shared folders](#mounting-custom-shared-folders)
-- [Networking](#networking)
-- [Persistent volumes](#persistent-volumes)
-- [Private container registries](#private-container-registries)
 
 <!-- /MarkdownTOC -->
 
-<a name="managing-your-openshift-instance"></a>
-## Managing your OpenShift instance
+<a name="managing-minishift"></a>
+## Managing Minishift
+When you use Minishift, you actually interact with:
+- the virtual machine(VM) created by Minishift and
+- the OpenShift instance provisioned by the VM.
 
-This section contains information about basic virtual machine and OpenShift management operations.
+This section contains information about managing the Minishift VM. For details on managing your local OpenShift instance using Minishift refer to the [Interacting with OpenShift](#interacting-with-openshift) and the [Configuring OpenShift](./configuring-openshift.md) sections.
 
-<a name="starting-openshift"></a>
-### Starting OpenShift
+<a name="minishift-start"></a>
+### Minishift start
 
-The [`minishift start`](./minishift_start.md) command is used to start your OpenShift instance.
-This command creates and configures a virtual machine that runs a single-node OpenShift instance.
+The [`minishift start`](./minishift_start.md) command is used to start a local OpenShift instance.
+This command creates and configures the Minishift VM that runs a single-node OpenShift instance.
 
-<a name="stopping-openshift"></a>
-### Stopping OpenShift
+It also copies the `oc` binary to your host so that you can interact with OpenShift either through the `oc` command line tool or through the web console which can be accessed through the URL provided in the output of the `minishift start` command. For further information refer to the [Interacting with OpenShift](#interacting-with-openshift) and [Configuring OpenShift](./configuring-openshift.md) sections.
```

Comment:
there is a specific command to get the URL, that should be used, instead of referring to some output of the start commands which will disappear. 

    $ minishift console --url